docs/en/setup/envoy/als_setting.md

```diff
@@ -11,6 +11,12 @@ SkyWalking is the first open source project introducing this ALS based solution
 
 You need three steps to open ALS.
 1. Open envoyAccessLogService in istio by [enabling **envoyAccessLogService** in ProxyConfig](https://istio.io/docs/reference/config/istio.mesh.v1alpha1/#ProxyConfig).
+
+    Upper istio 1.6.0, if istio installed by demo profile, you can open ALS ues command:
```

Comment:
I prefer to mention that the skywalking oap service is at `skywalking` namespace, and the port of gRPC service is `11800`